### PR TITLE
@types/dockerode - Improve typings for createVolume

### PR DESCRIPTION
**Pull request by userlerueda**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.docker.com/engine/api/v1.41/#operation/VolumeCreate
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
Github Actions service is facing issue, hence GA didn't ran in this PR
ref: https://www.githubstatus.com/

File: types/dockerode/index.d.ts
Comment (MysteryBlokHed) [inline types/dockerode/index.d.ts:469]: Based on the linked docs, it looks like `Status` is a map of key/value pairs and not a string

```suggestion
Status?: { [key: string]: string } | undefined;
```

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by MysteryBlokHed** [comment]
It looks like the PR still needs approval from either one of the other owners or a maintainer

**Comment by MysteryBlokHed** [comment]
@Seikho @nlaplante @isac322 @LazarusX @meisenzahl @thegecko @CameronDiver @P4sca1 @stuartthomson
Could one of you take a look at this PR? I'm not able to approve it since I'm not an owner and it's marked as a popular package.

**Review by thegecko — APPROVED** [review]

**Comment by userlerueda** [comment]
Ready to merge

**Pull request merged**
